### ISSUE TITLE
Notify when bookmarked tracks disappear

### ISSUE THREAD

**Issue by infernix** [cbeyls/fosdem-companion-android#24]
As per subject, when a cancelled talk disappears from bookmarks, please notify me so I am aware of the schedule change. Since it's already bookmarked I think this shouldn't be too hard?

**Comment by cbeyls** [comment]
I will take a look at how I can do this. Talks may also disappear when FOSDEM resets their database, so maybe you only want to be noticed for talks in the future?

**Comment by infernix** [comment]
Yes, that makes sense. it's relevant up to and including the days of the event but not for past events.

Which also implies that during the event, updates every 15/30/60 minutes would be good to have so that I'm informed as soon as possible.

**Comment by cbeyls** [comment]
The XML file is quite big, that's why the app is designed to work offline and the updates are never automatic. The app would consume way too much data with auto updates.

Also, the server does not properly handle the X-Modified-Since HTTP header properly anymore because the schedule file is auto-regenerated automatically all the time even when there are no changes, so the app could not even take advantage of this to update only when there are actual changes.

**Comment by cbeyls** [comment]
Can you tell me which talk has disappeared? Canceled talks are not supposed to be actually deleted from th DB, they should be updated with a big "CANCELLED" in their title. If that happened this is definitely a mistake from the person who managed the talk data.

**Comment by infernix** [comment]
IIRC it was https://fosdem.org/2017/schedule/event/alerting_with_time_series/ which was initially scheduled earlier in the day

**Comment by cbeyls** [comment]
So this is a different case: the talk has not been deleted but re-scheduled. Because of the way the database is updated (first clears everything, the re-store the entire DB), this is going to be hard to change.
